en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md

```diff
@@ -0,0 +1,1282 @@
+# Upgrading API Manager from 2.6.0 to 4.1.0
+
+The following information describes how to upgrade your API Manager server **from APIM 2.6.0 to 4.1.0**.
+
+!!! note
+    Before you follow this section, see [Upgrading Process]({{base_path}}/install-and-setup/upgrading-wso2-api-manager/upgrading-process) for more information.
+
+!!! Attention
+    If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, first you have to follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.11.0]({{base_path}}/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-570-to-is-5110) instead of below steps.
+
+!!! Attention
```

Comment:
Remove

Reply:
fixed in https://github.com/wso2/docs-apim/commit/d4a4773a2657877cd9ad081106301a4d29c1695c